### PR TITLE
Added information about impermanet loss risk in pool feature

### PR DESCRIPTION
**Pull request by AshishGapat**
Added a note about impermanent losses, as described in issue #1122 

image

### CODE REVIEW

**Comment by moodysalem** [comment]
There is also https://github.com/Uniswap/uniswap-interface/pull/1135

**Comment by AshishGapat** [comment]
Ah, did not see that. Never mind

**Comment by callil** [comment]
@AshishGapat I added a comment in #1135 that is relevant here.

**Comment by AshishGapat** [comment]
Refactored the copy as discussed in #1135 (credit for the copy goes to @jochenboesmans)

image

**Comment by 3esmit** [comment]
Excellent!

**Comment by AshishGapat** [comment]
Thanks @3esmit! 😊 
Anything else I need to do? I see that the lint check is failing but I'm not sure how to fix that

**Comment by moodysalem** [comment]
> Thanks @3esmit! 😊
> Anything else I need to do? I see that the lint check is failing but I'm not sure how to fix that

@callil can merge if it looks good

lint error is just due to this
https://github.com/wearerequired/lint-action/issues/13

File: src/pages/Pool/index.tsx [hunk 1]
```diff
@@ -132,6 +132,21 @@ export default function Pool() {
...
+                <TYPE.white fontWeight={600}>Liquidity provider risks</TYPE.white>
+              </RowBetween>
+              <RowBetween>
+                <TYPE.white fontSize={14}>
+                  {`Liquidity providers assume the risk of impermanent loss. It's possible to end up with less value than your initial input.`}
+                </TYPE.white>
+              </RowBetween>
+              <ExternalLink
+                style={{ color: 'white', textDecoration: 'underline' }}
+                target="_blank"
```
Comment (moodysalem) [inline src/pages/Pool/index.tsx:145]: target="_blank" is the default for `ExternalLink` component

**Review by moodysalem — APPROVED** [review]

**Comment by callil** [comment]
Sorry to delay this - will pull down and review on monday :) 

I think we can compress the message a little bit more.

**Comment by 3esmit** [comment]
Put a `(read more)` in smaller font size at right side of the title.

**Comment by AshishGapat** [comment]
Hi, sorry for the delay guys. I've made the change you described @3esmit. Does this look okay?
 
image

**Comment by moodysalem** [comment]
This is stale and is on the old v2 UI only, so just closing it for now

**Pull request closed without merging**